src/Command/InfectionCommand.php

```diff
@@ -103,6 +104,13 @@ protected function configure()
                 InputOption::VALUE_REQUIRED,
                 'Custom configuration file path'
             )
+            ->addOption(
+                'coverage',
```

Comment:
if it's path maybe we should call it `coverage-path` ?

Reply:
I thought the same but made it consistent with the `--configuration` options https://infection.github.io/guide/command-line-options.html#configuration-or-c which also does not have `path` postfix